CMakeLists.txt

```diff
@@ -51,12 +61,24 @@ osvr_add_plugin(NAME com_osvr_LeapMotion
 
 # If you use other libraries, find them and add a line like:
 target_link_libraries(com_osvr_LeapMotion
-	${LEAP_SDK_PATH}/lib/x86/Leap.lib
-	${OpenCV_DIR}/opencv_core2410.lib)
-
-add_custom_command(
-	TARGET com_osvr_LeapMotion
-	POST_BUILD
-	COMMAND cp $(TargetPath) ${CMAKE_PREFIX_PATH}/bin/osvr-plugins-0/com_osvr_LeapMotion.dll
-	COMMENT "Copying plugin build to OSVR Server..."
-)
+	Leap::Leap
+	${OpenCV_LIBS})
+
+# osvr_DIR points to OSVR binaries folder (built from source or downloaded binaries)
+# if you have OSVR built from source, it would have issues executing command below
+# just change the path accordingly
+if(WIN32)
```

Comment:
(Just have to set the CMAKE_INSTALL_PREFIX in CMake to be the same directory you installed OSVR to.)